pmm-app/src/shared/components/Elements/Table/Table.styles.ts

```diff
@@ -66,5 +66,13 @@ export const getStyles = stylesFactory((theme: GrafanaTheme) => {
     checkboxColumn: css`
       width: 20px;
     `,
+    tableClassicExplain: css`
+    table {
+      td {
+        min-inline-size: 4rem;
```

Comment:
why not min width?

Reply:
The min-inline-size property defines the horizontal or vertical minimal size of an element's block, depending on its writing mode. It corresponds to either the min-width or the min-height property, depending on the writing-mode. 
So, I chose it to set min 4rem for the direction of the lines of text to fit the string 'NULL' as the word that could be shown without line braking.  As the 'NULL' could be appeared in any column we do not need to set a minimum for each column separately.